src/sentry/static/sentry/app/views/settings/projectPlugins/projectPlugins.tsx

```diff
@@ -68,9 +78,6 @@ class ProjectPlugins extends Component {
 
           {plugins
             .filter(p => {
-              if (DEPRECATED_PLUGINS.includes(p.id) && !p.enabled) {
```

Comment:
The deprecated plugins of `taiga`, `lighthouse`, and `grove-io` have been long removed so this code is unnecessary.

Reply:
Can we remove `DEPRECATED_PLUGINS` elsewhere too?

Reply:
@billyvg done

Reply:
Nice